lib/prompts/accountNamePrompt.ts

```diff
@@ -56,7 +76,7 @@ const hubspotAccountNamePrompt = ({ accountType, currentPortalCount = 0 }) => {
     {
       name: 'name',
       message: promptMessageString,
-      validate(val) {
+      validate(val: string): string | boolean {
```

Comment:
With the `PromptValidationFunction` type, it doesn't seem like I should need to type this, but I get an error otherwise. Is that ok?